gating/pre_merge_test/run

```diff
@@ -33,6 +33,11 @@ if [[ $RE_JOB_ACTION == "tox-test" ]]; then
 elif [[ $RE_JOB_IMAGE =~ .*mnaio.* ]]; then
   bash -c "$(readlink -f $(dirname ${0})/run_deploy_mnaio.sh)"
   source "${MNAIO_VAR_FILE}"
+  # run tempest tests on MNAIO after deploy
+  ${MNAIO_SSH} <<EOS
+    cd /opt/openstack-ansible
+    openstack-ansible playbooks/os-tempest-install.yml
```

Comment:
This will only install it, not run it - unless you have ``tempest_run: yes`` set somewhere? To make it also run you can add ``-e tempest_run=yes`` to this command.